packages/x-charts/src/internals/store/useStore.ts

```diff
@@ -1,19 +1,20 @@
-import * as React from 'react';
-import { ChartsContext } from '../context/InteractionProvider';
-import { ChartStore } from './plugins/utils/ChartStore';
+import { useChartContext } from '../../context/ChartProvider';
+import { ChartStore } from '../plugins/utils/ChartStore';
+import { UseChartInteractionSignature } from '../plugins/featurePlugins/useChartInteraction/useChartInteraction.types';
 
-export function useStore(skipError?: boolean): ChartStore {
-  const charts = React.useContext(ChartsContext);
+// This hook should be removed because user and us should not interact with the store directly, but with public/private APIs
```

Comment:
When should we remove it?

Reply:
When the interaction fully move to the plugins, which would allow to remove the  hook for axis listener, and voronoi handler which are using the store to update interaction state.